### DISCUSSION TITLE
Support for Canadian Solar Inverters

### DISCUSSION THREAD

**Discussion by eddyann** [StephanJoubert/home_assistant_solarman#488] [Q&A]
![inverter](https://github.com/StephanJoubert/home_assistant_solarman/assets/82745167/816fef4a-e825-41f4-af3b-d3810b06c128)

How to enable the sensors for canadian solar inverter? The webserver GUI is similar to the documentation in the Solarman Manual, but I still not able to make it work. IP address and serial number are good

**Comment by eddyann** [comment]
![portcan](https://github.com/StephanJoubert/home_assistant_solarman/assets/82745167/58da549c-c50f-4204-b046-8434c00eb5cd)

does this required to send out data to homeassistant? or the port should be 8123? For now the HA cannot see the inverter. GUI web is fine

**Comment by Bibabakker** [comment]
Before the SOLARMAN app I use this link:
https://community.home-assistant.io/t/collect-inverter-data-from-solarman-api/328760

The problem of this methode is I have to refresh the API every 30 days.

anyone already find a way to make te SOLARMAN app work voor the cannadian solar inverter?

**Comment by tok1satsu1** [comment]
I have this same inverter. Is there any news?

**Comment by xpeppe86x** [comment]
What are the webserver page for Canadian solar? only IP addess?
